ui/src/message_popup/author/questions_page.jsx

```diff
@@ -59,18 +66,37 @@ export default React.createClass({
     return questionString.split(text).length;
   },
   
+  onQuestionsReceived(err, response){
+    const allDatabaseQuestions = JSON.parse(response.text).row;
+    if(allDatabaseQuestions === undefined) {
```

Comment:
It was initially there to set the question lists if the database was not connected. I do see now that it could be set in `getInitialState` and overwritten when getting the data from the database.